pubspec.lock

```diff
@@ -1335,18 +1367,18 @@ packages:
     dependency: transitive
     description:
       name: sentry
-      sha256: "19a267774906ca3a3c4677fc7e9582ea9da79ae9a28f84bbe4885dac2c269b70"
+      sha256: "57514bc72d441ffdc463f498d6886aa586a2494fa467a1eb9d649c28010d7ee3"
```

Comment:
Why is the Sentry package changed?

Reply:
I think my flutter dependency resolver updated version: "7.20.0" to version: "7.20.2"